public/walkthroughs/markdown-walkthrough/walkthrough.adoc

```diff
@@ -0,0 +1,277 @@
+# A custom walkthrough written in markdown
```

Comment:
@aidenkeating @david-martin  here's some thinking about markdown support.

Reply:
A few q's.

* Is there anything special the webapp has to do/include to support this (anything to supplement asciidoctor.js)?
* Is it likely *we* will be writing any walkthrough content in markdown?
* What is the benefit of supporting markdown?
* Is it worth trying to support both asciidoc & markdown?

Reply:
@david-martin 
- asciidoctor supports some markdown by default, no changes required
- no, however many people resist asciidoc as a format, this just eases that conversion
- the story becomes 'this is a variant of markdown', not a new language you must learn
- no, and I'm not sure whether it works as a method of introducing asciidoc, but I feel you are the wrong audience for that test ;)